File: python/026. Remove Duplicates from Sorted Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) == 0: return 0
        cnt = 1
        for i in range(1,len(nums)):
            if nums[i] != nums[i-1]:
                nums[cnt] = nums[i]
                cnt += 1
        return cnt

File: C++/026. Remove Duplicates from Sorted Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int j=0;
        int size = nums.size();
        if(size<=1)
        {
            return size;
        }
        for(int i=0;i<size-1;i++)
        {
            if(nums[j]<nums[i+1])
            {
                nums[j+1]=nums[i+1];
                ++j;
            }
        }
        return j+1;
    }
};
